understanding/22/consistent-help.html

```diff
@@ -75,10 +74,9 @@ <h2>Benefits of Consistent Help</h2>
          <li>This Success Criterion helps people who may have difficulty locating help that is not part of the page they are using. Ensuring they can connect with help allows them to complete the task.</li>
          <li>When a user can easily find help for their specific questions, they are more successful completing tasks.</li>
          <li>Users that experience cognitive fatigue or cognitive shut down will be able to reserve their energy for the task, instead of using it to find support.</li>
-         <li>Enabling users (especially those with cognitive disabilities) to find solutions while expressing their question using their own words increases their chances of success for completing a task.</li>
+         <li>Enabling users (especially those with cognitive disabilities) to find solutions while expressing their question using their own words (for example by interacting with a <q>chatbot</a>) increases their chances of success for completing a task.</li>
```

Comment:
`<q>chatbot</a>` probably needs to be `<q>chatbot</q>` (unless you're linking to a defined term, in which case change the opening tag to `<a>`)